From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sun, 10 Jan 2010 16:04:18 +0100
Subject: [PATCH] Dots are special in groff

so don't start a line with it or the rest of the line will be dropped.

---
 multipath/multipath.conf.5 |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/multipath/multipath.conf.5 b/multipath/multipath.conf.5
index 33b8747..a2864c6 100644
--- a/multipath/multipath.conf.5
+++ b/multipath/multipath.conf.5
@@ -207,9 +207,9 @@ If set to \fIpriorities\fR the multipath configurator will assign
 path weights as "path prio * rr_min_io". Possible values are
 .I priorities
 or
-.I uniform
-. Default is
-.I uniform
+.IR uniform .
+Default is
+.IR uniform .
 .TP
 .B no_path_retry
 Specify the number of retries until disable queueing, or
-- 
